src/zampy/recipe.py

```diff
@@ -62,9 +62,10 @@ def __init__(self, recipe_filename: str) -> None:
         # Load & parse recipe
         recipe = recipe_loader(recipe_filename)
 
-        start_year, end_year = recipe["download"]["years"]
+        self.start_year, self.end_year = recipe["download"]["years"]
         self.timebounds = TimeBounds(
-            np.datetime64(f"{start_year}"), np.datetime64(f"{end_year}")
+            np.datetime64(f"{self.start_year}-01-01T00:00"),
+            np.datetime64(f"{self.end_year}-12-13T23:59"),
```

Comment:
why `-12-13`? shouldn't be `-12-31`? :thinking:

Reply:
I am not sure about hardcoding the time format here  `np.datetime64(f"{self.start_year}-01-01T00:00")`. What will happen if the data has a different format? Shall we add a comment here to explain it?

Reply:
Oh yeah. That should be 31.

The reason I did this is that the recipe currently has `years = [start, end]` as format. We can change this to be specific date+time strings.